scripts/releases/upload-release-assets-for-dotslash.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+const {REPO_ROOT} = require('../shared/consts');
+const {getWithCurl} = require('./utils/curl-utils');
+const {
+  processDotSlashFileInPlace,
+  validateDotSlashArtifactData,
+} = require('./utils/dotslash-utils');
+const {
+  FIRST_PARTY_DOTSLASH_FILES,
+} = require('./write-dotslash-release-asset-urls');
+// $FlowFixMe[untyped-import] TODO: add types for @octokit/rest
+const {Octokit} = require('@octokit/rest');
+const nullthrows = require('nullthrows');
+const path = require('path');
+const {parseArgs} = require('util');
+
+const config = {
+  allowPositionals: true,
+  options: {
+    token: {type: 'string'},
+    releaseId: {type: 'string'},
+    force: {type: 'boolean', default: false},
+    dryRun: {type: 'boolean', default: false},
+    help: {type: 'boolean'},
+  },
+};
+
+async function main() {
+  const {
+    positionals: [version],
+    values: {help, token, releaseId, force, dryRun},
+    /* $FlowFixMe[incompatible-call] Natural Inference rollout. See
+     * https://fburl.com/workplace/6291gfvu */
+  } = parseArgs(config);
+
+  if (help) {
+    console.log(`
+  Usage: node ./scripts/releases/upload-release-assets-for-dotslash.js <version> --release_id <id> --token <github-token> [--force]
+
+  Scans first-party DotSlash files in the repo for URLs referencing assets of
+  an upcoming release, and uploads the actual assets to the GitHub release
+  identified by the given release ID.
+
+  If run with --force, the script will overwrite any assets that happen to
+  already exist at the given URLs. This is useful for retrying failed or
+  corrupted uploads.
```

Comment:
maybe let's add a short description of the parameters...
- what is version? which shape can it have? (if it is the react-native version, should start with `v`or not?